cbmc/proofs/DeserializeConnack/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags:        '=--object-bits;7;--unwind;0;--unwinding-assertions;--32;--bounds-check;--pointer-check='
```

Comment:
Does the (large) space carry particular meaning?

Reply:
This is just a tab from the macro definition used to generate the string.